docs/advanced-features/s3-prefix-permissions.md

```diff
@@ -0,0 +1,34 @@
+# Restricting S3 access to specific prefix in bucket
+
+> User will still have access to the full lists of S3 objects,
```

Comment:
"User" change to "Users"
"Full lists" change to "Full list" (remove the "s")

Reply:
I suggest making suggestions using `suggestion` feature. See item 6 in https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request.